src/__mocks__/graceful-fs.js

```diff
@@ -170,7 +170,12 @@ fs.__setMockFilesystem = function(object) {
 };
 
 function getToNode(filepath) {
-  var parts = filepath.split('/');
+  // Ignore the drive for Windows paths.
+  if (filepath.charAt(1) === ':') {
```

Comment:
maybe just regex over whether the first character is a letter and the second one is a `:`? :)